BuildingSync.xsd

```diff
@@ -14777,6 +14803,21 @@ Window spacing: the dimension between windows in a discrete window layout. (in.)
 			</xs:restriction>
 		</xs:simpleType>
 	</xs:element>
+	<xs:element name="BurnerControlType">
+		<xs:annotation>
+			<xs:documentation>Control type of burner, if applicable.</xs:documentation>
+		</xs:annotation>
+		<xs:simpleType>
+			<xs:restriction base="xs:string">
+				<xs:enumeration value="Full Modulation Set to Manual"/>
+				<xs:enumeration value="Full Modulation Set to Modulation"/>
```

Comment:
@supriyagoel Same here, we're not so sure about this name either.

Reply:
These options are from NYC LL87 audit data collection form. Based on literature I found online, there are 3 predominant type of controls-
1. On Off
2. High Fire/Low Fire (specified as High Low in BSXML)
3. Modulating

I believe the LL87 form provides 3 options for the modulating control (Step Modulation, Full Modulation- Set to Manual, Full Modulation- Set to Modulating)

BSXML could either use the options from LL87 or the 3 control types listed above (along with an option for 'Unknown')